### ISSUE TITLE
Regarding Resampling

### ISSUE THREAD

**Issue by vivekaxl** [automl/auto-sklearn#449]
Short version: I would like to pass the train and test datasets (for holdout)---which can be used instead of random splits as defined by ‘resampling_strategy_arguments.’ Is it possible in the current version?

Longer version: 
Consider the example: 
```
>>> import autosklearn.classification
>>> cls = autosklearn.classification.AutoSklearnClassifier()
>>> cls.fit(X_train, y_train)
```
cls would split the data into 67:33 and perform Algorithm Configurations. It is possible to explicity pass the train and test data for the holdout? 

Please correct me if I am wrong.

**Comment by mfeurer** [comment]
No, unfortunately, this is not possible yet. @EnricoTesta works on this in PR #442 and this is very close to being merged into the development branch.

**Comment by mfeurer** [comment]
I just merged the PR to the development branch and this should work now in the development branch. Please re-open if there are any further issues.

**Comment by vivekaxl** [comment]
Thank you, @mfeurer.

**Comment by cse387** [comment]
Hello
According to the initial thread 
do we know whether we could specify manually train-test splits disabling any random resampling strategy ?

If we specify the following args in the class and fit function, is that enough?

clf = autosklearn.classification.AutoSklearnClassifier(
    time_left_for_this_task=36,
    per_run_time_limit=3,
    # resampling_strategy='holdout',
    # resampling_strategy_arguments={'train_size': len(train_indx), 'test_size': len(test_indx)},
    metric = accuracy)

automl.fit(X= X[train_indx,:][:,train_indx], y= y[train_indx], X_test= X[train_indx,:][:,train_indx], y_test= y[train_indx])

Thanks
